pulsar-broker/src/test/java/org/apache/pulsar/broker/zookeeper/ZooKeeperClientAspectJTest.java

```diff
@@ -73,7 +73,9 @@
     
     static {
         // load agent with aspectjweaver-Agent for testing
-        AgentLoader.loadAgentClass(Agent.class.getName(), null);
+        // maven-test waves advice on build-goal so, maven doesn't need explicit loading
+        // uncomment it while testing on eclipse: 
+        //AgentLoader.loadAgentClass(Agent.class.getName(), null);
```

Comment:
I couldn't find a way to find out that advice is already registered. I will fix it, once figure out the way.